manifests/0000_31_cluster-baremetal-operator_03_baremetalhost.crd.yaml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Nit. Can we remove this line?

Reply:
It's just copied from https://github.com/metal3-io/baremetal-operator/blob/master/config/crd/bases/metal3.io_baremetalhosts.yaml

I can remove it manually but IMHO we should fix this CRD update process to avoid any manual edits  (e.g put any downstream stuff into `get-bmh-crd.sh` but otherwise just copy from either upstream or the openshift BMO fork.